db/migrate/20160630082357_drop_editions_summary.rb

```diff
@@ -0,0 +1,5 @@
+class DropEditionsSummary < ActiveRecord::Migration
+  def change
+    remove_column :editions, :summary
```

Comment:
The code gets checked out, the migrations are run and then the app instances are restarted.